dolphinscheduler-worker/src/test/java/org/apache/dolphinscheduler/server/worker/runner/operator/TaskInstanceOperationFunctionTest.java

```diff
@@ -0,0 +1,292 @@
+
+        TaskInstanceDispatchOperationFunction taskInstanceDispatchOperationFunction =
+                new TaskInstanceDispatchOperationFunction(
+                        workerConfig,
+                        workerTaskExecutorFactoryBuilder,
+                        workerTaskExecutorThreadPool);
+
+        try (MockedStatic<LogUtils> logUtilsMockedStatic = Mockito.mockStatic(LogUtils.class)) {
+            logUtilsMockedStatic
+                    .when(() -> LogUtils
+                            .getTaskInstanceLogFullPath(any(TaskExecutionContext.class)))
+                    .thenReturn("test");
+            TaskInstanceDispatchResponse taskInstanceDispatchResponse = taskInstanceDispatchOperationFunction.operate(
+                    new TaskInstanceDispatchRequest(taskExecutionContext));
+            Assertions.assertEquals(taskInstanceDispatchResponse.isDispatchSuccess(), false);
+            logUtilsMockedStatic.verify(times(1), () -> LogUtils.removeWorkflowAndTaskInstanceIdMDC());
+
+            given(workerTaskExecutorThreadPool.submitWorkerTaskExecutor(any())).willReturn(true);
+            taskInstanceDispatchResponse = taskInstanceDispatchOperationFunction.operate(
+                    new TaskInstanceDispatchRequest(taskExecutionContext));
+            Assertions.assertEquals(taskInstanceDispatchResponse.isDispatchSuccess(), true);
+            logUtilsMockedStatic.verify(times(2), () -> LogUtils.removeWorkflowAndTaskInstanceIdMDC());
+        }
+    }
+
+    @Test
+    public void testTaskInstanceKillOperationFunction() {
+        TaskInstanceKillOperationFunction taskInstanceKillOperationFunction = new TaskInstanceKillOperationFunction(
+                workerManager,
+                messageRetryRunner);
+
+        try (MockedStatic<LogUtils> logUtilsMockedStatic = Mockito.mockStatic(LogUtils.class)) {
+            int taskInstanceId = 111;
+            logUtilsMockedStatic
+                    .when(() -> LogUtils
+                            .setTaskInstanceLogFullPathMDC(any(String.class)))
+                    .then(invocationOnMock -> null);
+            TaskInstanceKillResponse response = taskInstanceKillOperationFunction.operate(
+                    new TaskInstanceKillRequest(taskInstanceId));
+            Assertions.assertEquals("Cannot find WorkerTaskExecutor", response.getMessage());
+        }
+
+        try (MockedStatic<LogUtils> logUtilsMockedStatic = Mockito.mockStatic(LogUtils.class)) {
+            int taskInstanceId = 111;
+            TaskExecutionContext context = new TaskExecutionContext();
+            context.setLogPath("logpath");
+            logUtilsMockedStatic
+                    .when(() -> LogUtils
+                            .setTaskInstanceLogFullPathMDC(any(String.class)))
+                    .then(invocationOnMock -> null);
+            taskInstanceKillOperationFunction.operate(
+                    new TaskInstanceKillRequest(taskInstanceId));
+        }
```

Comment:
It seems that there is no test in this try block

Reply:
It looks like this part is no longer needed after refactor.
I've removed this part.